src/test/java/random_csapatnev/modelclasses/FieldTest.java

```diff
@@ -0,0 +1,261 @@
+    	//Arrange
+    	Bear bear2 = new Bear("bear");
+    	Bear bear3 = new Bear("bear");
+    	Bear bear4 = new Bear("bear");
+
+    	field.getCharacters().add(bear2);
+    	field.getCharacters().add(bear3);
+    	field.getCharacters().add(bear4);
+    	field.getCharacters().add(bearMock);
+    	
+    	//Act
+        field.bearInteract(bearMock);
+
+        //Assert
+        verify(bearMock, times(0)).characterInteract(any(Bear.class));
+    }
+	
+	@Test
+	public void bearInteract__DoesNotCallCharacterInteract_WhenCharactersListIsEmpty() {
+		// Act
+		field.bearInteract(bearMock);	// Nincs karakter a mezőn
+
+	    // Assert
+	    verify(bearMock, never()).characterInteract(any(Character.class));
+	}
+	
+	@Test
+	public void removeCharacter_RemovesCharacterFromField() {
+	    // Arrange
+	    field.getCharacters().add(character1);
+	
+	    // Act
+	    field.removeCharacter(character1);
+	
+	    // Assert
+	    assertFalse(field.getCharacters().contains(character1));
+	}
+
+	 @Test
+	 public void removeCharacter_DoesNothing_WhenCharacterNotPresent() {
+        // Arrange
+        field.getCharacters().add(character1);
+
+        // Act
+        field.removeCharacter(character2);
+
+        // Assert
+        assertTrue(field.getCharacters().contains(character1));
+	 }
+	
+	
+	@Test
+	public void moveFrom_AddsCharacterToCurrentField() {
+		// Arrange
+	    Field field2 = new Field(0,1);
+	    field2.getCharacters().add(character1);
+	
+	    // Act
+	    field.moveFrom(field2, character1);
+	
+	    // Assert
+	    assertTrue(field.getCharacters().contains(character1));
+     }
+	 
+	 @Test
+	 public void moveFrom_AddsMultipleCharactersToCurrentField() {
+        // Arrange
+        Field field2 = new Field(0,1);
+        field2.getCharacters().add(character1);
+        field2.getCharacters().add(character2);
+
+        // Act
+        field.moveFrom(field2, character1);
+        field.moveFrom(field2, character2);
+
+
+        // Assert
+        assertTrue(field.getCharacters().contains(character1));
+        assertTrue(field.getCharacters().contains(character2));
+     }
+	 
+	 @Test
+	 public void moveFrom_RemovesCharacterFromOtherField() {
+        // Arrange
+        Field field2 = new Field(0,1);
+        field2.getCharacters().add(character1);
+
+        // Act
+        field.moveFrom(field2, character1);
+
+        // Assert
+        assertFalse(field2.getCharacters().contains(character1));
+     }
+	 
+	 @Test
+	 public void moveFrom_RemovesMultipleCharacterFromOtherField() {
+        // Arrange
+        Field field2 = new Field(0,1);
+        field2.getCharacters().add(character1);
+        field2.getCharacters().add(character2);
+
+        // Act
+        field.moveFrom(field2, character1);
+        field.moveFrom(field2, character2);
+
+        // Assert
+        assertFalse(field2.getCharacters().contains(character1));
+        assertFalse(field2.getCharacters().contains(character2));
+     }
+	 
+	 /*@Test
```

Comment:
Kommentelt kódot vagy vedd ki, vagy kommenteld oda azt is, hogy miért hagyjuk bent.